### PR TITLE
Add i.fourtf.com as an image uploader example

### PR DESCRIPTION
**Pull request by alazymeme**
Pull request checklist:

- [x] `CHANGELOG.md` was updated, if applicable

# Description

Adds fourtf's image uploader as an example for users that want to host their own 🚀

### CODE REVIEW

File: docs/IMAGEUPLOADER.md [hunk 1]
```diff
@@ -45,3 +45,13 @@ Replace `XXXXXXXXXXXXXXX` with your API key from s-ul.eu. It can be found on [yo
 |Extra headers||
 |Image link|`{url}`|
 |Deletion link|`https://s-ul.eu/delete.php?file={filename}&key=XXXXXXXXXXXXXXX`|
+
+### [i.fourtf.com](https://github.com/fourtf/i)
+
```
Comment (zneix) [inline docs/IMAGEUPLOADER.md:50]: I think you could add some kind of information explaining that this is an example config for websites/servers hosting this project, not an actual uploading service.

File: docs/IMAGEUPLOADER.md [hunk 2]
```diff
@@ -45,3 +45,13 @@ Replace `XXXXXXXXXXXXXXX` with your API key from s-ul.eu. It can be found on [yo
...
+|Row|Description|
+|-|-|
+|Request URL|`https://i.yourwebsite.com/upload`|
+|Form field|`file`|
+|Extra headers|`Authorization: Basic <Your base64 encoded user and password>`|
```
Comment (zneix) [inline docs/IMAGEUPLOADER.md:55]: Replace `<Your base64 encoder user and password>` with something way shorter, like `XXXXXXXXXXXXXXX` (from an example above) with information to replace that with authentication.

**Review by zneix — CHANGES_REQUESTED** [review]

**Comment by alazymeme** [comment]
Need to change this align with https://github.com/Chatterino/wiki/pull/6

**Comment by fourtf** [comment]
doesn't make sense in this repo, if you add it to the wiki repo then I'll merge

**Comment by alazymeme** [comment]
https://github.com/Chatterino/wiki/pull/8

**Pull request closed without merging**